tests/suite/cli_misc.rs

```diff
@@ -541,6 +541,22 @@ fn run_rls_when_not_installed() {
     });
 }
 
+#[test]
+fn run_rls_when_not_installed_for_nightly() {
+    setup(&|config| {
+        config.expect_ok(&["rustup", "default", "stable"]);
+        config.expect_ok(&["rustup", "install", "nightly"]);
```

Comment:
```suggestion
        config.expect_ok(&["rustup", "toolchain", "install", "nightly"]);
```

Reply:
Are we already emitting a warning about this when the command is executed?

Reply:
> Are we already emitting a warning about this when the command is executed?

I believe we do:

https://github.com/rust-lang/rustup/blob/7790fae8cd49a1c55b6251fab1d5bdde3c5b05ab/src/cli/rustup_mode.rs#L68-L85

https://github.com/rust-lang/rustup/blob/7790fae8cd49a1c55b6251fab1d5bdde3c5b05ab/src/cli/rustup_mode.rs#L168

Reply:
@djc Last time I checked, this interface has remain unchanged for years and `rustup toolchain install` has evolved away from it (so they do slightly different things now), but the simplest form still has a bunch of users.

Reply:
> Curious how/why @smoelius decided to include it in this test.

Not sure what you meant, so here are two answers.

Why that syntax? Just because that is what I am accustomed to using.

Why install the nightly toolchain? Because without it, one gets a "toolchain ... is not installed" error.